integration-commons/src/main/java/org/symphonyoss/integration/Integration.java

```diff
@@ -28,7 +29,7 @@
  *
  * Created by Milton Quilzini on 04/05/16.
  */
-public interface Integration {
+public interface Integration extends HealthIndicator {
```

Comment:
@dnsymphony Makes sense. I removed the dependency from HealthIndicator interface and created an adapter on integration-healthcheck module to get the health information from Integration interface and adapts to Health interface required by Spring Boot Actuator.